records_mover/records/cli.py

```diff
@@ -114,7 +116,17 @@ def main() -> None:
     args = parser.parse_args()
     raw_config = vars(args)
     func = getattr(args, 'func', None)
-    if func is None:
+    if args.healthcheck:
+        try:
+            engine = AirbyteEngine()
+            result = engine.healthcheck()
+            if result:
+                print("Airbyte Status: OK!")
+            else:
+                print("Airbyte Status: Unhealthy")
+        except Exception:
+            sys.exit(1)
```

Comment:
can we instead let the exception propagate so it's visible to the user? it'll sys exit anyway